### DISCUSSION TITLE
Beta disgarded?

### DISCUSSION THREAD

**Discussion by AdamMiltonBarker** [community/community#19066] [Copilot Conversations]
My copilot notified me I needed to update settings today, so just to confirm, after helping with the beta-program we are now blocked from using copilot unless we pay ?

**Comment by rauzerfabiano** [comment]
Unfortunately yes... from August 22nd it will be paid. I don't agree at all either, we helped and fed the copilot, we gave them millions of lines of code and now they want to charge us.

**Reply by AdamMiltonBarker** [reply]
Yer last time I help with any MS betas.

**Comment by 201flaviosilva** [comment]
It's stupid to have to pay for a tool you learned thanks to open source projects
